src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { Todo } from '../../Types/Todo';
+import { User } from '../../Types/User';
+
+interface Props {
+  onAdd: (todo: Todo) => void;
+  users: User[];
+  max: number;
+}
+
+export const NewTodo: React.FC<Props> = ({ onAdd, users, max }) => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [id, setId] = useState(max);
+  const [completed, setCompleted] = useState(false);
+
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [hasUserError, setHasUserError] = useState(false);
+
+  const onTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setHasTitleError(false);
+  };
+
+  const onUserIdChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+    setHasUserError(false);
+  };
+
+  const newTodo: Todo = {
+    id: max + 1,
```

Comment:
Using `max + 1` for the new todo's `id` could lead to duplicate IDs if `max` is not updated correctly after adding a new todo. Consider using a more reliable method to generate unique IDs.